examples/server/server.c

```diff
@@ -1576,10 +1565,6 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
                 usePskPlus = 1;
                 break;
 
-            case 'n' :
```

Comment:
Please also remove `n` from `mygetopt_long` call above. It is now, available.

Reply:
Will do.  My goal is to get this diff to 10k lines!